### DISCUSSION TITLE
How do I limit the number of episodes displayed in the Show page?

### DISCUSSION THREAD

**Discussion by Hubro** [pymedusa/Medusa#9659] [Q&A] [answered]
I have a show with something like 750 episodes over 19 seasons. Any time this page opens, it freezes or lags horribly for around 20 seconds while the page loads. After Googling around a bit, it seems like it should be possible to limit the number of loaded episodes, but after scouring the Medusa settings pages several times I can't find any such option. "Table pagination" under interface settings is enabled, but it doesn't seem to have any effect.

Is this possible to do?

**Comment by p0psicles** [comment, marked as answer]
At the bottom of the page you can configure the pagination

**Reply by Hubro** [reply]
Which page do you mean? The bottom of the show page looks like this:

![image](https://user-images.githubusercontent.com/597206/122996791-8fc73e80-d3ab-11eb-99e9-7d38346e618c.png)

I don't see any pagination options :confused:

**Reply by Hubro** [reply]
Aaah alright, I found the options, they were below the seasons but above the specials. Since the show I'm watching has hundreds of specials that means the pagination options were quite high up on the page. Unfortunately the pagination settings don't seem to affect specials though. I set "Rows per page" to 25, so now it's displaying 25 episodes and almost 300 specials. :thinking:

**Reply by medariox** [reply]
You can hide the specials. Look top right on the show page.

**Reply by Hubro** [reply]
Aah, and the setting is saved per show! Awesome!

Thanks a bunch!
